css/src/components/form/radio.scss

```diff
@@ -0,0 +1,96 @@
+$radio-circle-size: 1.125rem !default;
+$radio-dot-size: 0.75rem !default;
+$radio-dot-checked-color: $primary !default;
+$radio-dot-color: $text-subtle !default;
+$radio-timing-function: $input-timing-function !default;
+$radio-duration: $input-transition-duration !default;
+$radio-animation: boop !default;
+$radios-spacing: 0.5rem !default;
+
+@mixin radio-dot {
+	position: absolute;
+	inset: 0;
+	width: $radio-dot-size;
```

Comment:
Could just be my screen, but I see varying degrees of dot wonkiness on different zooms. Not entirely sure it's avoid able, but could you double check?

![image](https://user-images.githubusercontent.com/30843002/157937453-3848e5ae-87a1-421d-8b67-2fb673ae3e2c.png)

Reply:
I thought it went away for me for a while, but today it seems to have returned. I drew inspiration from a codepen that our colleague Scott O'Hara created, which doesn't seem to have this problem but haven't figured out why. 

https://codepen.io/scottohara/pen/MWoEGZe

Reply:
Looks to me as well that the Figma has larger dots relative to circle size than what's implemented, although it's pretty close

https://www.figma.com/file/uVA2amRR71yJZ0GS6RI6zL/%F0%9F%8C%9E-Atlas-Design-Library?node-id=838%3A1096

Reply:
Updated the dot size